main.py

```diff
@@ -12,6 +12,7 @@
 import subprocess
 
 import github_api
+from github_api import get_pull_requests_for_issue
```

Comment:
I think we need to define get_pull_requests_for_issue in github_api.py. And then we should use it like 'github_api.get_pull_requests_for_issue' rather than importing it from github_api.

Reply:
@Aiderbot I think we need to define get_pull_requests_for_issue in github_api.py. And then we should use it like 'github_api.get_pull_requests_for_issue' rather than importing it from github_api.

Reply:
@Aider can you try this again? We need to define get_pull_requests_for_issue in github_api.py. And then we should use it like 'github_api.get_pull_requests_for_issue' rather than importing it from github_api.

Reply:
@Aiderbot can you try this again? We need to define get_pull_requests_for_issue in github_api.py. And then we should use it like 'github_api.get_pull_requests_for_issue' rather than importing it from github_api.

Reply:
@Aiderbot can you try this again? We need to define get_pull_requests_for_issue in github_api.py. And then we should use it like 'github_api.get_pull_requests_for_issue' rather than importing it from github_api.